xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1635,11 +1635,11 @@ bool CGUIWindowVideoBase::GetDirectory(const std::string &strDirectory, CFileIte
     newPlaylist->m_bIsFolder = true;
     items.Add(newPlaylist);
 
-/*    newPlaylist.reset(new CFileItem("newplaylist://", false));
```

Comment:
Well one is touching the music library and the other the video library. And the "newplaylist://" can not only be used for musicvideos IIRC so this is not strictly a music video only change.